ui/raidboss/data/07-dt/alliance/cloud_of_darkness_chaotic.ts

```diff
@@ -0,0 +1,673 @@
+        left: {
+          en: 'Left, under hand',
+        },
+        right: {
+          en: 'Right, under hand',
+        },
+        aero: {
+          en: 'Knockback',
+        },
+        death: Outputs.outThenIn,
+        out: Outputs.out,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Deluge of Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: ['9E3D', '9E01'], source: 'Cloud of Darkness', capture: false },
+      response: Responses.bleedAoe('alert'),
+    },
+    {
+      id: 'Cloud Chaotic Grim Embrace Collector',
+      type: 'StartsUsing',
+      netRegex: { id: ['9E39', '9E3A'], source: 'Cloud of Darkness', capture: true },
+      run: (data, matches) => {
+        data.deadlyEmbraceDodgeDirectionCollected = matches.id === '9E39' ? 'back' : 'front';
+      },
+    },
+    {
+      id: 'Cloud Chaotic Deadly Embrace',
+      type: 'GainsEffect',
+      netRegex: { effectId: '1055', capture: true },
+      condition: Conditions.targetIsYou(),
+      preRun: (data) =>
+        data.deadlyEmbraceDodgeDirection = data.deadlyEmbraceDodgeDirectionCollected,
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 7,
+      durationSeconds: 7,
+      infoText: (data, _matches, output) => output[data.deadlyEmbraceDodgeDirection]!(),
+      outputStrings: {
+        back: {
+          en: 'Bait hand, dodge backwards',
+        },
+        front: {
+          en: 'Bait hand, dodge forwards',
+        },
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Rapid-sequence Particle Beam',
+      type: 'StartsUsing',
+      netRegex: { id: '9E40', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Wild Charge (alliances)',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Unholy Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: 'A12D', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.healerGroups!(),
+      outputStrings: {
+        healerGroups: Outputs.healerGroups,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Flare Marker',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.flareMarker, capture: true },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.flare!(),
+      outputStrings: {
+        flare: {
+          en: 'Flare on you',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Break IV',
+      type: 'StartsUsing',
+      // 9E4F = dummy cast from boss (ends slightly before actual cast)
+      // 9E50 = actual cast from boss (sometimes has stale source name)
+      // 9E51 = dummy cast from adds (source: Sinister Eye)
+      // 9E52 = actual cast from adds (sometimes has stale source name)
+      netRegex: { id: '9E4F', source: 'Cloud of Darkness', capture: false },
+      response: Responses.lookAway(),
```

Comment:
I haven't looked through the logs too hard, but is it going to be possible to math out where the adds are and then call one of the safe intercardinals for the player to look at? If so, maybe we could put that in the TODO list?

Reply:
It might be possible, but it would be more delayed than the current trigger.  It looks like the Sinister Eyes spawn in off-map to the east, then move into location right before they start casting.